bcda/models/models.go

```diff
@@ -265,6 +270,32 @@ func AssignAlphaBeneficiaries(db *gorm.DB, aco ACO, acoSize string) error {
 	return db.Exec(s).Error
 }
 
+type CCLF8File struct {
+	gorm.Model
+	Name            string
+	ACOCMSID        string `gorm:"column:aco_cms_id"`
+	Timestamp       time.Time
+	PerformanceYear int
+}
+
+func (*CCLF8File) TableName() string {
+	return "cclf8_files"
+}
+
+// "The MBI has 11 characters, like the Health Insurance Claim Number (HICN), which can have up to 11."
+// https://www.cms.gov/Medicare/New-Medicare-Card/Understanding-the-MBI-with-Format.pdf
+type CCLF8Beneficiary struct {
+	gorm.Model
+	FileID        uint
```

Comment:
I think this is a "belongs to" relationship and should have this format:
File CCLF8File (or CCLFMetadata)
FileID uint

Reply:
The beneficiaries do belong to a file, but I don't know that it's necessary to make a relationship to that type here just yet, before we know how we're using these models. Would having the file as a field of `CCLF8Beneficiary` impact the database tables or constraints? There is a foreign key for `cclf8_beneficiaries.file_id` to `cclf8_files.id`.

Reply:
I think we need both.  I haven't done it in practice, but I think it would let us do this

db.Model(&user).Related(&profile)
from http://doc.gorm.io/associations.html
or this:
// Query - Find out all related associations
db.Model(&user).Association("Languages").Find(&languages)
from http://doc.gorm.io/associations.html#association-mode

Maybe we can just add it in when the need arises though.